cli/azd/pkg/apphost/generate.go

```diff
@@ -90,6 +90,12 @@ func init() {
 					return strings.ReplaceAll(src, ".", "")
 				},
 				"envFormat": scaffold.EnvFormat,
+				"bicepParameterValue": func(value *string) string {
+					if value == nil {
+						return ""
+					}
+					return fmt.Sprintf(" = '%s'", *value)
```

Comment:
Just double-checking here: we're only supporting strings for now? I think the single quotes need omitted for things like bool, int, object and array.